app/models/service_template_ansible_playbook.rb

```diff
@@ -132,7 +132,7 @@ def self.validate_config_info(info)
 
 
   def job_template(action)
-    resource_actions.find_by(:action => action.to_s.capitalize).try(:configuration_template)
+    resource_actions.find_by(:action => action.capitalize).try(:configuration_template)
```

Comment:
With the latest changes I'm not able to get as far as before.

Using the following scenario:

1. Create new Ansible Playbook catalog item with no retirement playbook.
2. Edit the catalog item and add a new retirement playbook.
3. Edit the catalog item and remove the retirement playbook.
4. Delete the entire catalog item.

I am failing at 2.  The retirement resource_action is created:

<img width="1413" alt="screen shot 2017-05-05 at 3 26 13 pm" src="https://cloud.githubusercontent.com/assets/697347/25761021/4fb2e614-31a7-11e7-8c52-e942124e5422.png">

But the retirement playbook is not created in Tower:

<img width="1420" alt="screen shot 2017-05-05 at 3 27 35 pm" src="https://cloud.githubusercontent.com/assets/697347/25761057/70c58fe6-31a7-11e7-91c4-a5a07125272e.png">